defaults/main.yml

```diff
@@ -159,8 +159,13 @@ openio_oioswift_filter_container_hierarchy:
   account_first: true
   strip_v1: true
   swift3_compat: true
-  create_dir_placeholders: true
-  recursive_placeholders: true
+#  create_dir_placeholders: true
+#  recursive_placeholders: true
+  sentinel_hosts:
+    "{{ groups['openio_redis_cluster'] \
+    | map('extract', hostvars, ['openio_bind_address'])\
+    | join(':6012,') }}:6012"
+  sentinel_name: "{{ openio_oioswift_sds_proxy_namespace }}-master-1 "
```

Comment:
Use `openio_oioswift_namespace` instead of `openio_oioswift_sds_proxy_namespace`